### DISCUSSION TITLE
Ability to add custom fields to the edit order page

### DISCUSSION THREAD

**Discussion by bossanova808** [craftcms/commerce#2416] [Ideas]
**What are you trying to do?**

I'd really like to be able to add custom fields on our order to the main order screen.

Craft 3.5 brought very welcome improvements to field layout, but it is still not possible to put fields - such as e.g. tracking fields, or custom notes fields, on the main order page.

This means, often, that quite important information is buried behind tabs, and easily missed.

**What's your proposed solution?**

Allow the addition of fields to main order page.  Even if as a somewhat separate thing above or below (as I understand the main thing is a Vue app now...)

**Additional context**

It's all about increasing the reliability and ease of order processing.  Customer fields on orders are a great Commerce feature, but they are not living up to their full potential because - for really important fields - access is not as easy as it should be.

**Comment by lukeholder** [comment]
We closed #865 since it was referring to 'fields' which you can now add (custom forms etc). 

This ticket is asking for custom fields to be places on the first tab of the order edit page. This is not something we are going to do just yet, but keeping it on the list for the future.
